ast/compile.go

```diff
@@ -1277,6 +1284,39 @@ func (c *Compiler) getExports() *util.HashMap {
 	return rules
 }
 
+func (c *Compiler) checkDuplicateImports() {
+	if !c.strict {
+		return
+	}
+
+	for _, name := range c.sorted {
+		mod := c.Modules[name]
+		processedImports := map[Var]*Import{}
+
+		for _, imp := range mod.Imports {
+			var name Var
+
+			if len(imp.Alias) > 0 {
```

Comment:
https://github.com/open-policy-agent/opa/blob/main/ast/policy.go#L647 Would `(Import).Name()` be useful here? 🤔

Reply:
Cool! Yes it would 👍